src/cosmicqc/frame.py

```diff
@@ -591,24 +604,87 @@ def find_image_columns(self: CytoDataFrame_type) -> bool:
         ]
 
     @staticmethod
+    def draw_outline_on_image(actual_image_path: str, mask_image_path: str) -> Image:
+        """
+        Draws outlines on a TIFF image based on a mask image and returns
+        the combined result.
+
+        This method takes the path to a TIFF image and a mask image, creates
+        an outline from the mask, and overlays it on the TIFF image. The resulting
+        image, which combines the TIFF image with the mask outline, is returned.
+
+        Args:
+            actual_image_path (str): Path to the TIFF image file.
+            mask_image_path (str): Path to the mask image file.
```

Comment:
Will it matter if the mask is a TIFF file or will this function require the mask be a specific file type?

Reply:
Thanks, great question, this work is specific to TIFF's for the time being. I imagine adjustments will need to be made for other image types.